### PR TITLE
fix AHP_slow_time and min_AHP_values

### PR DESCRIPTION
**Pull request by AurelienJaquier**
## Description

`min_AHP_values` and `AHP_slow_time` are a little bit special because they are computed during other features computation (`min_AHP_indices` and `AHP_depth_abs_slow`). So when we call them, the feature function does nothing, except calling the other feature with the dependency graph. The problem was that the 'empty' feature function still returns `-1`, and that can be caught afterwards, and replace the feature values with `None` (depending on the order of the features passed to efel).

This PR just replaces these `-1` with `1`s. The feature is no longer replaced by `None`. In the case where the feature cannot be computed but still returns `1`, this does not cause a problem, because when `pyfeatures.cppfeature_access.get_cpp_feature` will not find the feature, it will just replace it with `None`, as expected.

## Checklist:
- [ ] Unit tests are added to cover the changes (skip if not applicable).
- [ ] The changes are mentioned in the documentation (skip if not applicable).
- [ ] CHANGELOG file is updated (skip if not applicable).

### CODE REVIEW

**Comment by AurelienJaquier** [comment]
replace empty list features with None, following what is described here: Issue #321

File: efel/cppcore/cfeature.cpp [hunk 1]
```diff
@@ -303,7 +303,10 @@ int cFeature::getFeature(string strName, vector<T>& vec) {
       return -1;
     }
     vec = getMapData<T>(strName, *dataMap);
-    if (vec.empty()) GErrorStr += "Feature [" + strName + "] data is missing\n";
+    if (vec.empty()) {
+      GErrorStr += "Feature [" + strName + "] data is missing\n";
+      return -1;
```
Comment (AurelienJaquier) [inline efel/cppcore/cfeature.cpp:308]: This will turn empty lists into `None`

**Comment by ilkilic** [comment]
wouldn't be clearer to call the dependent feature by using `getFeatures` and return their returning values instead?

**Comment by AurelienJaquier** [comment]
How would we do that concretely?

**Review by ilkilic — APPROVED** [review]
Fantastic! Thank you!

**Pull request merged**